.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: cpp
+branches:
+  only:
+  - coverity_scan 
+
+env:
+  global:
+    secure: "heviCPZymO/3impAjVchArRIjP/UdWdIapTkK8eX1KL+dDNNJiDDXCGi9ZCHoZW1CzIEc3QWcG7jn4wyPesMiDuMsGcvZFIVMx1E1KQNJ+D2Wu+EST9bRicAx+7knEosEVVFeWbXtL3QqVtza3qQ1EhryOUz6dMGIvoIZkLhWiI="
+
+before_install:
+- sudo sh -c "echo 'deb http://download.opensuse.org/repositories/isv:/ownCloud:/desktop/xUbuntu_12.04/
+  /' >> /etc/apt/sources.list.d/owncloud-client.list"
+- sudo sh -c "echo 'deb-src http://download.opensuse.org/repositories/isv:/ownCloud:/desktop/xUbuntu_12.04/
+  /' >> /etc/apt/sources.list.d/owncloud-client.list"
+- wget http://download.opensuse.org/repositories/isv:ownCloud:desktop/xUbuntu_12.04/Release.key
+- sudo apt-key add - < Release.key
+- sudo apt-get update
+- sudo apt-get build-dep owncloud-client
+- checkout=$(git show-ref --head --hash head)
+- cd ../
+- wget https://scan.coverity.com/download/linux-64 --post-data "token=$token&project=owncloud%2Fmirall" -O coverity_tool.tgz
+- mkdir coverity
+- tar -xvf coverity_tool.tgz -C coverity --strip-components=1
+- export PATH=$PATH:$PWD/coverity/bin/
+- cd $TRAVIS_BUILD_DIR
+
+install:
+- cd ../
+- mkdir client-build
+- cd client-build
+- cmake -DCMAKE_BUILD_TYPE="Debug" $TRAVIS_BUILD_DIR
+- cov-build --dir cov-int make
+- tar czvf client.tgz cov-int 
+- curl --form token=$token --form email=lukas@statuscode.ch --form file=@$PWD/client.tgz --form version="$checkout" --form description="$checkout" https://scan.coverity.com/builds?project=owncloud%2Fmirall
+
+# Hack to stop processing
+script: time
```

Comment:
This is a huge hack since it will otherwise call `make` and so on. Shouldn't matter though since if a step fails before it will fail before that and if it reaches this line all is fine…